Account/templates/Account/profile.html

```diff
@@ -76,16 +72,12 @@ <h3 style="color: rgb(99, 99, 99);">@{{user.username}}</h3>
       <div style="width: 30%; float: right;"> 
         <form method="post" action="{% url 'Account:profile' user.username %}">
           {% csrf_token %}
-            {% if not index %}
-            <button name="follow{{user.username}}" type="submit" style="float:right" value={{user.username}} class="follow_button">Follow{{ list_follow.j }}</button>
-            {% else %}
-            <button name="unfollow{{user.username}}" type="submit" style="float:right" value={{user.username}} class="follow_button">Unfollow</button>
-            {% endif %}        
+            <button name="follow" type="submit" style="float:right" value={{user.username}} class="follow_button">Follow</button>
```

Comment:
'Who to follow' section should have have anyone that we already follow...so removed 'Unfollow' button